kinbot/stationary_pt.py

```diff
@@ -143,7 +143,13 @@ def bond_mx(self):
         for i in range(self.natom):
             for j in range(self.natom):
                 if i == j: continue
-                if self.dist[i][j] < constants.st_bond[''.join(sorted(self.atom[i]+self.atom[j]))]:
+                atom_pair = [self.atom[i], self.atom[j]]
+                atom_pair = sorted(atom_pair)
+                print(atom_pair)
+                #print("i: {} j: {}".format(self.atom[i], self.atom[j]))
+                #print("joined: {}".format(''.join(sorted(self.atom[i]+self.atom[j]))))
+                #if self.dist[i][j] < constants.st_bond[''.join(sorted(self.atom[i]+self.atom[j]))]:
+                if self.dist[i][j] < constants.st_bond[''.join(atom_pair)]:
```

Comment:
The original code was printing "CCll" as the atom pair when Cl, Cl were the two atoms at i and j

Reply:
OK, because it was sorting the strings. You solved this somehow? I don't see it here how.

Reply:
the original code was reading in atom[i] and atom[j] and then combining them into a single string and then sorting the characters, which works fine for elements with a single character for their atomic symbol. For elements with 2 letters in their atomic symbol this fails (see example below). Instead I created an array of the two elements atomic symbols, sorted them and then joined them. 

For example
atom[i] = Cl
atom[j] = Cl
original code: 
`''.join(sorted(self.atom[i]+self.atom[j]))` --> joining characters C, l, C, l = `ClCl` then sorted into `CCll`

new code:
`atom_pair = [self.atom[i], self.atom[j]]` --> `['Cl', 'Cl']`
`atom_pair = sorted(atom_pair)` --> sorts the list above
`''.join(atom_pair)` --> joins together the strings `Cl` and `Cl` --> `ClCl`

Reply:
ok!